docs/source/api/apollo-server.mdx

```diff
@@ -228,6 +229,22 @@ Available in Apollo Server v3.4.0 and later.
 </td>
 </tr>
 
+<tr>
+<td>
+
+##### `cache`
+
+`KeyValueCache | "bounded"`
+</td>
+<td>
+
+It is recommended that you set this option to `"bounded"` if you otherwise would not have it configured. By default, the cache is unbounded. The APQ feature will use this cache by default (if you don't configure it otherwise). The default bounded cache is an [`InMemoryLRUCache`](https://www.npmjs.com/package/@apollo/utils.keyvaluecache) with a default size of roughly 30MiB. This can be configured by passing in your own cache instance.
```

Comment:
I think I would start by saying what this does. This is the way you configure a cache backend that can be used by many features: APQs, full response cache, Data Sources, etc. Notably, it is made available to all plugins.

Then after that say that the default is an unbounded cache; this default is not recommended because clients can cause servers to crash by filling memory with APQs. If you don't want to choose a specific implementation, we recommend `"bounded"`.

(And link to the new cache backend page but that's another PR right? So this can be left til that PR as long as it happens...)

Reply:
Good feedback, that's much better